lib/chef/resource/openssl_x509_certificate.rb

```diff
@@ -159,7 +159,6 @@ class OpensslX509Certificate < Chef::Resource
           owner new_resource.owner unless new_resource.owner.nil?
           group new_resource.group unless new_resource.group.nil?
           mode new_resource.mode unless new_resource.mode.nil?
-          sensitive true
```

Comment:
This should probably be set based on the "sensitive" property on the openssl_x509_certificate resource so people can still turn it on/off.

Reply:
In principle the certificate is never private. I'm not seeing a single use case where the certificate should be omitted from the output. What am I missing here?

Reply:
Yeah that's a fair take on it.